webapp/Pipfile

```diff
@@ -27,6 +27,8 @@ pydantic = "*"
 flask-static-digest = "*"
 python-dotenv = "*"
 pyhumps = "*"
+redislite = "*"
+fakeredis = "*"
```

Comment:
Please check:
https://digitalservice4germany.atlassian.net/wiki/spaces/STL/pages/269615183/Add+a+new+external+library

Reply:
Both redislite and fakereids fulfill our newly specified requirements right?